geomstats/geometry/matrices.py

```diff
@@ -26,8 +28,9 @@ def belongs(self, point):
         return mat_dim_1 == self.m & mat_dim_2 == self.n
 
     @staticmethod
-    def mul(a, b):
-        """Return the product of matrices a and b.
+    def equal(a, b, atol=TOLERANCE):
```

Comment:
Agree with @ninamiolane, for consistency these should be called `mat_a` and `mat_b`